packages/generate-prism-languages/index.ts

```diff
@@ -0,0 +1,110 @@
+import flowRight from "lodash.flowright"
+import pc from "picocolors"
+import { readFile, writeFile, access } from "node:fs/promises"
+import { constants } from "node:fs"
+import { join, dirname } from "node:path"
+import { languages as prismLanguages } from "prismjs/components"
+import uglify from "uglify-js"
+
+export const languagesToBundle = <const>[
+  "jsx",
+  "tsx",
+  "swift",
+  "kotlin",
+  "objectivec",
+  "rust",
+  "graphql",
+  "yaml",
+  "go",
+  "cpp",
+  "markdown",
+]
+
+/**
+ * We need to disable typechecking on this generated file as it's just concatenating JS code
+ * that starts off assuming Prism lives in global scope. We also need to provide Prism as that
+ * gets passed into an iffe preventing us from needing to use global scope.
+ */
+const header = `// eslint-disable-next-line @typescript-eslint/ban-ts-comment\n// @ts-nocheck\nimport Prism from "prismjs"\n`
+const prismPath = dirname(require.resolve("prismjs"))
+
+const readLanguageFile = async (language: string): Promise<string> => {
+  const pathToLanguage = join(prismPath, `components/prism-${language}.js`)
+  try {
+    await access(pathToLanguage, constants.R_OK)
+    const buffer = await readFile(pathToLanguage, { encoding: "utf-8" })
+    return buffer.toString()
+  } catch (e) {
+    return ""
+  }
```

Comment:
I'm mixed on if we should swallow the error here gracefully or terminate the script because a language file should not be missing from the upstream Prism package (ideally)